lib/streamlit/config_option.py

```diff
@@ -191,8 +192,10 @@ def __init__(
                 deprecation_text = "Replaced by %s." % self.replaced_by
 
         if self.deprecated:
-            assert expiration_date, "expiration_date is required for deprecated items"
-            assert deprecation_text, "deprecation_text is required for deprecated items"
+            if expiration_date is None:
+                raise ValueError("expiration_date is required for deprecated items.")
+            if deprecation_text is None:
+                raise ValueError("deprecation_text is required for deprecated items.")
```

Comment:
Thought/Nit: perhaps its more equivalent if these are `if not expiration_date` & `if not deprecation_text`... but since they are `str | None` and really not supposed to receive an empty string, don't feel strongly.

Reply:
Updated 👍